src/passes/analysis/verilog.cpp

```diff
@@ -1113,6 +1123,15 @@ bool Passes::Verilog::runOnInstanceGraphNode(InstanceGraphNode& node) {
     verilog_generators_seen.count(module->getGenerator()) > 0) {
     return false;
   }
+  if (is_mantle_wire(module)) {
+      // Check if all instances are inlined
+      bool all_inlined = false;
+      for (auto inst : node.getInstanceList()) {
+          all_inlined = check_inline_verilog_wire_metadata(inst);
```

Comment:
looks like `all_inlined` is just the last `inst`? should there be a `break` or something or an `|=` ?

Reply:
good catch